charts/values.yaml

```diff
@@ -0,0 +1,43 @@
+replicaCount: 1
+appsVars:
```

Comment:
Great job here!

Note the convention for doing this is breaking similar values into objects this way:

```
mongo:
  appname: mongo
  image: mongo
  tag: latest
  port: 27017
api:
  appname: api
  image: otomato/maskshop-api
  tag: latest
  port: 80
```
and then using the values as `{{ .Values.api.appname }}` etc.

Reply:
Copy that! thanks